source/Engine.cpp

```diff
@@ -354,16 +354,18 @@ void Engine::Place()
 		// taking off from a specific planet or nearby.
 		if(ship->GetSystem() == system && !ship->IsDisabled())
 		{
-			const Personality &person = ship->GetPersonality();
+			const Personality &personality = ship->GetPersonality();
 			bool hasOwnPlanet = ship->GetPlanet();
-			bool launchesWithPlayer = (planet && planet->CanLand(*ship))
-					&& !person.IsStaying() && !person.IsWaiting()
-					&& (!hasOwnPlanet || (ship->IsYours() && ship->GetPlanet() == planet));
+			bool launchesWithPlayer = (ship->GetPlanet() == planet
+					&& !personality.IsStaying() && !personality.IsWaiting());
```

Comment:
Doesn't this break NPCs without planets that should be launching with the player? I think it should be enough to restrict `hasOwnPlanet` for non-player ships.

Reply:
you mean NPCs with `launching` personality?

Reply:
No, NPCs without launching, without staying and without waiting.